docs/yarn_classic.md

```diff
@@ -48,32 +48,11 @@ Yarn Classic PM is capable of processing the following types of dependencies
 
 ## Fetching dependencies for Yarn Classic projects
 
-The process of fetching dependencies for Yarn Classic is similar to that for any
-other package manager. The name of package manager is `yarn_classic`, and it
-does not expect any additional arguments.
+The process of fetching dependencies for Yarn Classic is identical as for Yarn.
+But only `yarn` is a supported CLI option. Hermeto automatically detects Yarn
```

Comment:
> But only `yarn` is a supported CLI option. Hermeto automatically detects Yarn...

I think _Hermeto automatically detects Yarn Classic (i.e. v1) projects and so the logic is all conveniently hidden behind a single `yarn` CLI backend option._ might read slightly better.